src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -40,7 +40,7 @@ export function CreateExportModal({
   const [issuesToExport, setIssuesToExport] = useState<ExportableIssue[]>([]);
   const [startDate, setStartDate] = useState<Date | null>(null);
   const [endDate, setEndDate] = useState<Date | null>(null);
-
+  const [exportHovered, setexportHovered] = useState(false);
```

Comment:
```suggestion
  const [exportHovered, setExportHovered] = useState(false);
```
What about proper camel casing?